ios/Loki Network/LKDaemon.swift

```diff
@@ -0,0 +1,35 @@
+import Foundation
+
+final class LKDaemon {
+    
+    // MARK: Lifecycle
+    static let shared = LKDaemon()
+    
+    private init() { }
+    
+    // MARK: Configuration
+    func configure(isDebuggingEnabled: Bool = false, completionHandler: @escaping (Error?) -> Void) {
+        // Prepare
+        let directoryPath = FileManager.default.urls(for: .documentDirectory, in: .userDomainMask).first!.path
+        // Enable debugging mode if needed
+        if isDebuggingEnabled { _llarp_enable_debug_mode() }
+        // Generate configuration file
+        let configurationFilePath = directoryPath + "/" + "lokinet-configuration.ini"
+        _llarp_ensure_config(configurationFilePath, directoryPath, true, false)
+        // Download bootstrap file
+        let remoteBootstrapFilePath = "https://i2p.rocks/i2procks.signed"
```

Comment:
ideally this should be configurable if it's not already

Reply:
I thought about that, and there are other things that we might want to make configurable as well at some point, but I think it's a bit too early to be concerned with things like that right now.